dallinger/command_line/docker_ssh.py

```diff
@@ -360,6 +415,17 @@ def deploy(
     dashboard_user = config.get("dashboard_user", "admin")
     dashboard_password = config.get("dashboard_password", secrets.token_urlsafe(8))
 
+    if (
```

Comment:
@pmcharrison  We want to test early and not when deployment to the server already started. But I am not happy about putting this check here. Any idea how we could improve it?

Reply:
Yes, this is however only implemented in https://github.com/Dallinger/Dallinger/pull/6788
So, we want to merge that PR first.

Reply:
@pmcharrison Worked on this one in https://github.com/Dallinger/Dallinger/pull/7228/commits/7565bb4a914f709280b0764f286fd1bf78547ab4